src/settings-ui/Settings.UI/Assets/Settings/Scripts/EnableModule.ps1

```diff
@@ -26,15 +31,28 @@ if (!(Test-Path $PROFILE))
 $profileContent = Get-Content -Path $PROFILE -Raw
 
 if ((-not [string]::IsNullOrEmpty($profileContent)) -and ($profileContent.Contains("34de4b3d-13a8-4540-b76d-b9e8d3851756")))
+{
+  if ($profileContent.Contains("Import-Module `"$scriptPath\WinGetCommandNotFound.psd1`""))
+  {
+    $profileContent = $profileContent.Replace("Import-Module `"$scriptPath\WinGetCommandNotFound.psd1`"",
+                                              "Import-Module -Name Microsoft.WinGet.CommandNotFound")
+    $profileContent = $profileContent.Replace("34de4b3d-13a8-4540-b76d-b9e8d3851756",
+                                              "f45873b3-b655-43a6-b217-97c00aa0db58")
+    Set-Content -Path $PROFILE -Value $profileContent
+    Write-Host "Module was successfully upgraded in the profile file."
+    # This message will be compared against in Command Not Found Settings page code behind. Take care when changing it.
+  }
+}
+elseif ((-not [string]::IsNullOrEmpty($profileContent)) -and ($profileContent.Contains("f45873b3-b655-43a6-b217-97c00aa0db58")))
 {
   Write-Host "Module is already registered in the profile file."
   # This message will be compared against in Command Not Found Settings page code behind. Take care when changing it.
 }
 else
 {
-  Add-Content -Path $PROFILE  -Value "`r`n#34de4b3d-13a8-4540-b76d-b9e8d3851756 PowerToys CommandNotFound module"
-  Add-Content -Path $PROFILE  -Value "`r`nImport-Module `"$scriptPath\WinGetCommandNotFound.psd1`""
-  Add-Content -Path $PROFILE  -Value "#34de4b3d-13a8-4540-b76d-b9e8d3851756"  
+  Add-Content -Path $PROFILE  -Value "`r`n#f45873b3-b655-43a6-b217-97c00aa0db58 PowerToys CommandNotFound module"
+  Add-Content -Path $PROFILE  -Value "`r`nImport-Module -Name Microsoft.WinGet.CommandNotFound`""
```

Comment:
@carlos-zamora , have you tested this? (when the module is not installed and you install from scratch?) 
![image](https://github.com/microsoft/PowerToys/assets/26118718/18d51e81-9ba3-43d1-8868-341182a1a9cf)